odl/operator/default_ops.py

```diff
@@ -340,11 +360,29 @@ def adjoint(self):
         >>> op2 = MultiplyOperator(3.0, domain=r3, range=r3)
         >>> op2.adjoint(x)
         rn(3).element([ 3.,  6.,  9.])
+
+        Multipy operator with complex space:
```

Comment:
This reads as if you multiply an operator with a space. Maybe write "Multiplication operator"